### DISCUSSION TITLE
How to get Image support in DocumentAssembler to work?

### DISCUSSION THREAD

**Discussion by nefarius** [sergey-tihon/Clippit#52] [Q&A] [answered]
Hi 👋 

I'm interested in [this feature](https://sergey-tihon.github.io/Clippit/tutorials/word/DocumentAssembler_ImagesSupport.html) but regardless of using NuGet version [`1.10.2`](https://www.nuget.org/packages/Clippit/1.10.2) or [`2.0.0-beta2`](https://www.nuget.org/packages/Clippit/2.0.0-beta2) I can't produce a table row with an image in it. Unfortunately it seems I can't even get a valid document at all, I get this error on opening (and yes, the exact same code works fine with the original library version):

![image](https://user-images.githubusercontent.com/286631/166108033-652db676-351b-42c8-800c-178e9a0c88d7.png)

Here's an excerpt of the template:

![image](https://user-images.githubusercontent.com/286631/166108063-e6919b80-4112-4041-b573-6a8959f80f10.png)

Any insights appreciated! Cheers ✨

**Comment by nefarius** [comment, marked as answer]
**EDIT:** it's fixed! I had an empty cell in my table row, apparently that generates invalid markup, I simply added an empty string property to the template object and now it works! 🎉 

OK we got progress, turns out my template syntax was outdated, I used the documents used in the tests as a template and now the image is rendered, yay! Now I need to figure out what the "unreadable content" dialog still comes up, maybe there's more hidden broken stuff in the template itself...

Hm, I rewrote the template from scratch and still get the error. Currently DIFF'ing the generated file against one that got saved and "fixed" by MS Word.

**Reply by sergey-tihon** [reply]
Hi, glad you fixed the issue. 
Do you think you can [improve docs](https://sergey-tihon.github.io/Clippit/tutorials/word/DocumentAssembler_ImagesSupport.html) based on your experience?

**Reply by nefarius** [reply]
Definitely, I'll add i to my list 😃
